examples/development_lifecycle/development_lifecycle/decks.py

```diff
@@ -236,6 +236,30 @@ def table_renderer() -> None:
     )
 
 
+# %% [markdown]
+# :::{figure} https://raw.githubusercontent.com/flyteorg/static-resources/main/flytesnacks/user_guide/flyte_decks_source_code_renderer.png
```

Comment:
did you create a PR to static resources? I can't find it.

Reply:
sorry, I thought this image file would be generated automatically, right?

Reply:
it sounds like I need to manually create the file.

Reply:
updated
https://github.com/flyteorg/static-resources/pull/33